app/views/layouts/application.html.erb

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>SampleApp1</title>
-    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title><%= yield(:title) %> | Ruby on Rails Tutorial Sample App</title>
```

Comment:
`Ruby on Rails Tutorial Sample App` -> em tìm hiểu về `I18n in rails`
apply cho toàn bộ text dùng để hiển thị trên view cho ng dùng nhé 
-> đa ngôn ngữ nè

Reply:
.